bucket/tta.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://web.archive.org/web/20151027213538/http://en.true-audio.com",
+    "description": "TTA Lossless Audio Codec - command line frontend",
```

Comment:
This does not make sense. What this mean for end user?

Use something like `Command line wrapper for TTA Lossless Audio Codec`

Reply:
Just like `Free Lossless Audio Codec - command line tools` in `flac.json`.